app/services/twilio_service.rb

```diff
@@ -37,20 +44,11 @@ def twilio_client
     )
   end
 
-  def send_sms(params = {})
-    params = params.reverse_merge(from: from_number)
-    @client.messages.create(params)
-  end
-
   def from_number
     "+1#{account['number']}"
   end
 
-  def self.accounts
-    Rails.application.secrets.twilio_accounts
-  end
-
-  def self.random_account
-    accounts.sample
+  def random_account
+    JSON.parse(Figaro.env.twilio_accounts).sample
```

Comment:
Yes, but so will the memoized random_account variable if we were to add it there. Every time a new SMS is sent, we want to pick a random Twilio account, and we have to parse `application.yml` every time for that. What we don't want is to parse it again when `send_sms` is called on the TwilioService instance. Since `account` is called multiple times, we want to store its value (which corresponds to the random account) and reuse it for each instance.